rdmd.d

```diff
@@ -130,6 +131,7 @@ int main(string[] args)
             "eval", &eval,
             "loop", &loop,
             "exclude", &exclusions,
+            "include", (string opt, string p) { exclusions = exclusions.filter!(ex => ex != p).array(); },
```

Comment:
Hmm how much does this depend on the order of the arguments? I assume if you ran:

```
--exclude=abc --include=abc
```

abc would be included at the end. This is more of a question for getopt, since I'm not sure whether it iterates arguments left-to-right or right-to-left.